MantidQt/MantidWidgets/src/InstrumentView/RotationSurface.cpp

```diff
@@ -162,6 +162,7 @@ void RotationSurface::init() {
                             udet.u = applyUCorrection(udet.u);
                           }
                         }
+    updateViewRectForUCorrection();
```

Comment:
No, I tried to fix this in the previous PR but the code has detector info changes which I don't think we want in the patch release so I took that commit out.